### PR TITLE
kubectl v1.5.4 and SHA256 checksum

### LINKED ISSUE #14: Not clear where the kubectl binary is being downloaded to.
On the very first run you will see this message:

```
Downloading https://storage.googleapis.com/kubernetes-release/release/v1.4.6/bin/linux/amd64/kubectl.. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . OK
```

It's not clear where the binary is getting downloaded to, I think it should be clear to the users.

Also, in case you already have a `kubectl` binary in your path this step might be unnecessary.

### PR DESCRIPTION
**Pull request by hjacobs**
Fixes #14:

* Update to v1.5.4
* Add SHA256 checksum for download
* Add random suffix to support multiple downloads in parallel
* Allow setting the target folder via `KUBECTL_DOWNLOAD_DIR` env var (important for CI/CD environments where we want to have only one binary)

### CODE REVIEW

File: zalando_kubectl/main.py [hunk 1]
```diff
@@ -14,32 +18,40 @@
...
 KUBECTL_URL_TEMPLATE = 'https://storage.googleapis.com/kubernetes-release/release/{version}/bin/{os}/{arch}/kubectl'
-KUBECTL_VERSION = 'v1.5.3'
+KUBECTL_VERSION = 'v1.5.4'
+KUBECTL_SHA256 = '1d9be6531f9830798b94d2a3ba6d07592ac664b8e12a8ddf201dd815719b9563'
 
 
 def ensure_kubectl():
-    kubectl = os.path.join(click.get_app_dir(APP_NAME), 'kubectl-{}'.format(KUBECTL_VERSION))
+    path = Path(os.getenv('KUBECTL_DOWNLOAD_DIR') or click.get_app_dir(APP_NAME))
+    kubectl = path / 'kubectl-{}'.format(KUBECTL_VERSION)
```
Comment (mikkeloscar) [inline zalando_kubectl/main.py]: Is `/` some fancy path join operator? :)

Comment (hjacobs) [inline zalando_kubectl/main.py]: @mikkeloscar yes, see https://docs.python.org/3.5/library/pathlib.html

**Comment by mikkeloscar** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**